flasksrc/session.py

```diff
@@ -120,6 +129,15 @@ class ToyNetSessionByIdDeleteLinkPutResp(Schema):
     pass
 
 
+class ToyNetSessionByIdDeleteRouterInterfacePutReq(Schema):
+    name = fields.Str()
+    ip = fields.Str()
```

Comment:
Can you comment examples at least for the ip?

Reply:
Added a comment